tox.ini

```diff
@@ -55,6 +55,9 @@ python =
     pypy2: pypy2
     pypy3: pypy3
 
+[behave]
+paths = tests/acceptance
+
```

Comment:
HINT: I often need to disable that durations are shown (see: behave/behave.ini) to avoid regression when the test-run output is compared against its expectations. If there is currently no problem, its OK.